core/META-INF/MANIFEST.MF

```diff
@@ -4,28 +4,7 @@ Bundle-Name: Saros Core
 Bundle-SymbolicName: saros.core
 Bundle-Version: 0.2.0
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
-Export-Package: com.thoughtworks.xstream,
```

Comment:
Why do you remove the exports ?

Reply:
The xstream-Package is only used by the core and no other bundle. So the core don't need to export this package. If any other bundle is using this package and we missed to find it, please tell us that.